### DISCUSSION TITLE
Changed in eden but not loading in server

### DISCUSSION THREAD

**Discussion by Amateur-God** [Vdauphin/HeartsAndMinds#1068] [Question and answer]
Hi, Ive changed all the vehicles and players at the spawn point to Vehicles and Characters from the 3CB mods however when I load the mission up on the server and load in, everything is back to vanilla, do I need to changed any of the .sqf files for the change to take effect? I've checked the mission.sqm file over and over and when I load it in Eden they are all British, but when loaded on the server there back to vanilla

**Comment by Amateur-God** [comment]
The vehicles are now loading in, and if i give a character a custom loadout they load in with that loadout but all the modded assets on the character weapons etc all revert back to Vanilla

**Reply by Vdauphin** [reply]
Hello,
Just disable the loadout thing in mission paramter: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L356-L360
Cheers

**Reply by Amateur-God** [reply]
thank you, ive also switched the map over to takistan following your instructions, however i just get stuck on loading mission

**Reply by Vdauphin** [reply]
You did something wrong then

**Reply by Amateur-God** [reply]
i just followed this guide here to move to taklstan, however i did move the base and markers from where they spawned in to one of the air bases http://vdauphin.github.io/HeartsAndMinds/Change-MAP-of-Hearts-and-Minds

**Comment by Amateur-God** [comment]
I started again with a fresh download of the .altis mission file and tried again, it loads but for some reason doesn't spawn anyone in on takistan

**Reply by Vdauphin** [reply]
Does it work at least in Arma editor?

**Reply by Amateur-God** [reply]
No, if I load it as altis it works and spawns in people, but if I load it in takistan it doesn't

**Reply by Vdauphin** [reply]
Please follow steps here: http://vdauphin.github.io/HeartsAndMinds/Change-MAP-of-Hearts-and-Minds

**Reply by Amateur-God** [reply]
there the steps that i followed

**Reply by Vdauphin** [reply]
You didn't otherwise it would work
You just have to right click on your folder and replace .Altis to .Takistan
